pkg/apis/lighthouse.submariner.io/v2alpha1/doc.go

```diff
@@ -0,0 +1,4 @@
+// +k8s:deepcopy-gen=package
```

Comment:
I know it's weird, but let's jump back to v1alpha1, I think it's something we can afford at this stage.

Reply:
Specially since we are even changing resource names, etc.

Reply:
We agreed on slack, to be good and move to v2alpha